Pages/Student/Create.cshtml

```diff
@@ -1,47 +1,117 @@
+<style>
+ body{
+     background-image: url(https://banner2.cleanpng.com/20180326/lje/kisspng-polygon-geometry-plane-desktop-wallpaper-abstract-5ab9c141484b91.1154809315221230732961.jpg);
+     background-repeat: no-repeat;
+    background-size:100%;
+     
+}
+.my-form
+{
+    padding-top: 3.5rem;
+    padding-bottom: 2.5rem;
+}
+
+.my-form .row
+{
+    margin-left: 0;
+    margin-right: 0;
+}
+</style>
+</head>
+<body>
 <section>
-    <form method="post" asp-page="/Student/Create">
-        <div asp-validation-summary="All" class="text-danger"></div>
-        <div class="form-group">
-            <label asp-for="Input.Name"></label>
-            <input asp-for="Input.Name" class="form-control" />
-            <span asp-validation-for="Input.Name" class="text-danger"></span>
-        </div>
-        <div>
-            <label asp-for="Input.PRN"></label>
-            <input asp-for="Input.PRN" />
-            <span asp-validation-for="Input.PRN" class="text-danger"></span>
-        </div>
-        <div>
-            <label asp-for="Input.Email"></label>
-            <input asp-for="Input.Email" />
-            <span asp-validation-for="Input.Email" class="text-danger"></span>
-        </div>
-        <div>
-            <label asp-for="Input.MobileNo"></label>
-            <input asp-for="Input.MobileNo" />
-            <span asp-validation-for="Input.MobileNo" class="text-danger"></span>
-        </div>
-        <div>
-            <label asp-for="Input.Address"></label>
-            <input asp-for="Input.Address" />
-            <span asp-validation-for="Input.Address" class="text-danger"></span>
-        </div>
-        <div>
-            <label asp-for="Input.BirthDate"></label>
-            <input asp-for="Input.BirthDate" />
-            <span asp-validation-for="Input.BirthDate" class="text-danger"></span>
-        </div>
-        <div>
-            <label asp-for="Input.Year"></label>
-            <input asp-for="Input.Year" />
-            <span asp-validation-for="Input.Year" class="text-danger"></span>
-        </div>
+<main class="my-form">
+    <div class="cotainer">
+        <div class="row justify-content-center">
+            <div class="col-md-8">
+                    <div class="card">
+                        <div class="card-header">Register</div>
+                        <div class="card-body">
+                            <form name="my-form" method="post" asp-page="/Student/Create">
+                                <div asp-validation-summary="All" class="text-danger"></div>
+                                <div class="form-group row">
+                                    <label for="full_name" class="col-md-4 col-form-label text-md-right">Name</label>
+                                    <div class="col-md-6">
+                                        <input type="text" id="full_name" class="form-control" name="full-name">
```

Comment:
You shouldn't have removed original input field attributes such as ```asp-for```. They are needed for functioning and submission of forms. Same for all below input fields.